digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -307,6 +308,13 @@ public void cleanup(
         }
     }
 
+    private long maxWaitForFetchLogEventsInSec = 60L;
+
+    @VisibleForTesting
+    void setMaxWaitForFetchLogEvents(long sec) {
```

Comment:
`{` is supposed to be in the next line

Reply:
fixed https://github.com/treasure-data/digdag/pull/1603/commits/9bf8a447d7bb054e220cf7031a0da700074cadf7